taskcluster/ci/signing/kind.yml

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: focus_android_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - focus_android_taskgraph.transforms.multi_dep:transforms
+    - focus_android_taskgraph.transforms.signing:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - build
+
+primary-dependency: build
+
+group-by: build-type
+
+job-template:
+    description: Sign Focus for Android
+    worker: {}
+    signing-format: autograph_focus
+    index:
+        by-tasks-for:
+            (action|cron|github-release):
+                by-build-type:
+                    nightly:
+                        type: signing
+                    default: {}
+            default: {}
+    run-on-tasks-for: [github-push, github-pull-request]
```

Comment:
Eventually we'll want `github-release` here too, I think? It can probably be added now without causing any issuse, if you want. Definitely not blocking though.